src/Test/DatabaseResetter.php

```diff
@@ -34,11 +34,33 @@ public static function isDAMADoctrineTestBundleEnabled(): bool
         return \class_exists(StaticDriver::class) && StaticDriver::isKeepStaticConnections();
     }
 
+    public static function isResetUsingMigrations(): bool
+    {
+        return 'migrate' === ($_SERVER['FOUNDRY_RESET_MODE'] ?? null);
```

Comment:
```suggestion
        return 'migrate' === ($_SERVER['FOUNDRY_RESET_MODE'] ?? 'schema');
```

Maybe to be more explicit on what the default behavior is?